src/components/Team/ProfilePage/index.tsx

```
@@ -1,24 +1,27 @@
+import { zodResolver } from '@hookform/resolvers/zod';
 import { useQuery } from '@tanstack/react-query';
 import {
   teamRecruitmentProfileQueries,
   useUpsertTeamRecruitmentProfileMutation,
 } from 'api/teamRecruitmentProfile/queries';
-import LoadingSpinner from 'components/feedback/LoadingSpinner';
 import SubmitConfirmModal from 'components/Team/components/SubmitConfirmModal';
-import useTeamAuthGuard from 'components/Team/hooks/useTeamAuthGuard';
 import useTeamFormStep from 'components/Team/hooks/useTeamFormStep';
 import SubPageHeader from 'components/ui/SubPageHeader';
 import { FormProvider, useForm } from 'react-hook-form';
```

ESLint

[error] `react-hook-form` import should occur before import of `api/teamRecruitmentProfile/queries`

(import/order)